templates/CRM/Event/Form/EventFees.tpl

```diff
@@ -24,6 +24,7 @@
  +--------------------------------------------------------------------+
 *}
 {assign var=isRecordPayment value=1 }
+{assign var=isShowBillingBlock value=($action neq 2)}
```

Comment:
@eileenmcnaughton what is the action when value = 2?

Reply:
ok self learning looks like it means UPDATE as per https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/Action.php#L49